lib/src/common/cache/cache.dart

```diff
@@ -250,9 +269,21 @@ class Cache<TIdentifier, TValue> extends Object with Disposable {
   ///
   /// The decision of whether or not to actually remove the value will be up to
   /// the current [CachingStrategy].
+  ///
+  /// Release indicates that consuming code has finished with the [TIdentifier]
+  /// [TValue] pair associated with [id]. This pair may be removed immediately or
+  /// in time depending on the [CachingStrategy]. Access to the pair will be
+  /// blocked immediately, i.e. the [keys] and [values] getters won't report this
```

Comment:
> Access to the pair will be blocked immediately

Is this not a behavioral breakage?

Reply:
They would know better as they’re the ones using it, but it seems to me that the previous description of this API:

> The decision of whether or not to actually remove the value will be up to the current [CachingStrategy].

..implies that it _could_ be removed immediately, in which case consumers would also be unable to access it without another `.get()` or `.getAsync()` call. If that was the case, then I don’t see a behavioral difference because consumers would have had to guard against this scenario anyway.

Reply:
Sounds good to me 👍. Just making sure it was considered